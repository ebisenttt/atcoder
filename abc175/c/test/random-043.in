58867243766693 381008939541681 27199687201328
