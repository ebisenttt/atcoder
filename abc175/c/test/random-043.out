22731817837291
